client/src/components/Timeline.vue

```diff
@@ -2,20 +2,24 @@
     <canvas :id="datumid" height="50"></canvas>
 </template>
 <script>
-import Chart from 'chart.js';
-import moment from 'moment';
-import '../timeline.js';
+  import Chart from 'chart.js';
+  import moment from 'moment';
+  import '../timeline.js';
+  // TODO: Use proper method documentation format for JS
+// TODO: Remove unneeded props from this component
 export default {
   name: 'Timeline',
-  props: ['cdata', 'datumid', 'datumname'],
+  props: ['cdata', 'datumid', 'datumname','suntimes', 'timezone'],
```

Comment:
Where is the timezone used?

Reply:
It was initially used when I was trying to implement the sunrise/sunset annotations, but it is no longer used -- it has been removed in the latest commit.